src/utils/emulators.ts

```diff
@@ -30,5 +30,5 @@ export const signInEmulatorEmailPassword = async (
 }
 
 export const isAuthEmulatorEnabled = (): boolean => {
-    return Boolean(process.env.REACT_APP_AUTH_EMULATOR_ENABLED)
+    return process.env.REACT_APP_AUTH_EMULATOR_ENABLED === 'true'
```

Comment:
wasn't that more elegantly expressed with `Boolean` ?

Reply:
it work :D 
Boolean of the env always returned true in my case